README.md

```diff
@@ -1,42 +1,69 @@
-# What's inside
+# Containerized Ansible: What's inside #
 
-This container provide ansible toolstack inside a container.
+This container provides the ansible toolstack inside a container.
 * '''Dockerfile''' with the definition of the ansible container
-** based on OpenSUSE Tumbleweed
-** installs ansible and some additional tools
-* System Setup
-** Make sure needed packages installed on the host
----
-# sudo transactional-update pkg install podman python310-rpm
----
-*** Install ansible commands wrapers
-If current user is root /usr/local/bin is updated, for non-root users
-the user's ~/bin area is updated.
----
-as root:
-# (cd ~; podman container runlabel root-install ansible)
-as non-root
-# (cd ~; podman container runlabel user-install ansible)
----
-*** Ansible Commands
----
-ansible
-ansible-community
-ansible-config
-ansible-connection
-ansible-console
-ansible-doc
-ansible-galaxy
-ansible-inventory
-ansible-playbook
-ansible-pull
-ansible-vault
----
-
-*** Uninstall ansible commands wrapers
----
-as root:
-# (cd ~; podman container runlabel root-unstall ansible)
-as non-root
-# (cd ~; podman container runlabel user-unstall ansible)
----
+  * based on OpenSUSE Tumbleweed
+  * installs ansible and some additional tools
+## System Setup ##
+* Make sure needed packages installed on the host
+  * sudo transactional-update pkg install podman python310-rpm
+  * system reboot is required after all transactional updates
+    * sudo shutdown -r now
+
+## Ansible commands
+The ansible commands are provided as symlinks to ansible-wrapper.sh.
+The commands will instanciate the container and execute the ansible
+command.
+
+## To install ansible commands ##
+
+* as root:
+  * for the root user the ansible commands are placed in /usr/local/bin
+  * (cd /; podman container runlabel install ansible)
+* as non-root
+  * for non-root users the ansible commands are placed in the user's ~/bin
+  * (cd ~; podman container runlabel user-install ansible)
+
+## Ansible Commands ##
+* ansible
+* ansible-community
+* ansible-config
+* ansible-connection
+* ansible-console
+* ansible-doc
+* ansible-galaxy
+* ansible-inventory
+* ansible-playbook
+* ansible-pull
+* ansible-vault
+ 
+## Uninstall ansible commands  ##
+* as root:
+  * (cd /; podman container runlabel uninstall ansible)
+* as non-root
+  * (cd ~; podman container runlabel user-uninstall ansible)
+
+## Operation is through SSH back to container host or to other remote systems  ##
+Since ansible is running within a container, the local host is the container and not
+system instantiating the container. Any changes made to the local host are made to the
+container and would be lost when the container exits. To make host running the container
+host.containers.internal can be used in inventory record like:
+```
+inventory.yaml:
+all:
```

Comment:
We probably should change this here, and in the example, to look something more like:

```
alphost_group:
  hosts:
    alphost:
      ansible_host: host.containers.internal
```

I'll put up a PR to address that...

The reason for doing so is that technically `all` is a special group that is owned by the ansible inventory management system; any host that is defined in inventory gets automatically added to the `all` group.

We potentially could also define this in the /etc/ansible/hosts file in the container image by appending the following lines:

```
# Define the ALP podman host
[alphost_group]
alphost ansible_host=host.containers.internal
```

And then we just use `alphost` as a play target anywhere we need to run against the host.

Reply:
Have proposed #4 to address this.